subjects/leapyear/README.md

```diff
@@ -0,0 +1,39 @@
+## leap-year
+
+### Instructions
+
+Write a function named `LeapYear(int)` that takes a year as a parameter and returns true if the year is a leap year and false otherwise.
+- A leap year is a year divisible by 4, but not by 100.
+- A leap year is also divisible by 400.
```

Comment:
Did you  meant by the above statements the a Leap year in `only` divisible by 4

Reply:
Any number that can be divisible by 4 and not by 100 is a leap year. also if the number is divisible by 400 and divisible by 100 is a leap year

Reply:
- A Leap year is divisible by 4.
- A Leap year is divisible by  100 and 400.
- A leap year is not divisible by 100 only.
Is this right ?

Reply:
A Leap year is divisible by 4 and not 100 
A leap year is divisible by 4 and 100 and 400